consensus/babe/babe.go

```diff
@@ -45,18 +45,21 @@ type BabeSession struct {
 	txQueue *tx.PriorityQueue
 }
 
-// NewBabeSession returns a new Babe session using the provided VRF keys and runtime
-func NewBabeSession(pubkey VrfPublicKey, privkey VrfPrivateKey, rt *runtime.Runtime) *BabeSession {
-	return &BabeSession{
+// NewSession returns a new Babe session using the provided VRF keys and runtime
+func NewSession(pubkey VrfPublicKey, privkey VrfPrivateKey, rt *runtime.Runtime) *Session {
+	return &Session{
 		vrfPublicKey:  pubkey,
 		vrfPrivateKey: privkey,
 		rt:            rt,
-		txQueue:       new(tx.PriorityQueue),
 	}
```

Comment:
Where is this initialized now?

Reply:
woops accidentally deleted when merging, I've readded it